project/Dependencies.scala

```diff
@@ -23,16 +24,17 @@ object Dependencies {
       val playWS = groupId %% "play-ws" % exactPlayVersion % Provided
       val playTest = groupId %% "play-test" % exactPlayVersion % Test
 
-      // mockWs depends on some play-ahc-ws classes, so include them for tests
-      val playAhcWs = "com.typesafe.play" %% "play-ahc-ws" % exactPlayVersion % Test
-      val mockWs = "de.leanovate.play-mockws" %% "play-mockws" % mockWsVersion % Test
-      Seq(play, playWS, playTest, playAhcWs, mockWs)
+      Seq(play, playWS, playTest)
```

Comment:
Removing `play-mockws` and the associated tests in `GoogleAuthTest.scala`, as the `play-mockws` doesn't yet support Play 3 - we can reinstate them when https://github.com/leanovate/play-mockws/pull/322 is merged.